apps/queue-worker/src/worker.ts

```diff
@@ -0,0 +1,48 @@
+import * as Sentry from "@sentry/node";
+import { ProfilingIntegration } from "@sentry/profiling-node";
+import { NativeConnection, Worker } from "@temporalio/worker";
+import "dotenv/config";
+
+import * as activities from "@calcom/emails/email-manager";
+
+import { NAMESPACE, TASK_QUEUE_NAME } from "./shared";
+
+// eslint-disable-next-line turbo/no-undeclared-env-vars
+const { SENTRY_DSN } = process.env;
+if (SENTRY_DSN) {
+  Sentry.init({
+    dsn: SENTRY_DSN,
+    integrations: [new ProfilingIntegration()],
+    tracesSampleRate: 1.0,
+    profilesSampleRate: 1.0,
+  });
+  console.info("Sentry initialized");
+}
+
+async function run() {
+  const connection = await NativeConnection.connect({
+    address: "localhost:7233",
+    // TLS and gRPC metadata configuration goes here.
+  });
```

Comment:
This can be modified later to support local and production usage. It might be slightly different for the TLS settings if you decide to self-host or use the Cloud SaaS offering